packages/server/src/workers/subscription.ts

```diff
@@ -440,3 +440,65 @@ async function execBot(subscription: Subscription, resource: Resource): Promise<
 
   await createAuditEvent(resource, startTime, outcome, logResult, subscription, bot);
 }
+
+/**
+ * Sends a rest-hook subscription for deleted resource. Extra header is passed and the body is empty.
+ * @param job The subscription job details.
+ * @param subscription The subscription.
+ * @param resource The resource that triggered the subscription to add to the Audit Event.
+ */
+async function sendDeleteRestHook(
+  job: Job<SubscriptionJobData>,
+  subscription: Subscription,
+  resource: Resource
+): Promise<void> {
+  const url = subscription?.channel?.endpoint as string;
+  if (!url) {
+    // This can happen if a user updates the Subscription after the job is created.
+    logger.debug(`Ignore rest hook missing URL`);
+    return;
+  }
+
+  const startTime = new Date().toISOString();
+  const headers = buildRestHookHeaders(subscription, resource) as Record<string, string>;
```

Comment:
Casting this removes the error in line 465
```
Element implicitly has an 'any' type because expression of type '"X-Medplum-Deleted-Resource"' can't be used to index type 'HeadersInit'.
```